frame/wormhole/issuing/s2s/src/lib.rs

```diff
@@ -334,7 +332,7 @@ impl<T: Config> Pallet<T> {
 	pub fn judge_self_message(nonce: MessageNonce) -> DispatchResultWithPostInfo {
 		let message_sender = T::OutboundMessager::get_valid_message_sender(nonce)?;
 		ensure!(
-			T::ToEthAddressT::into_ethereum_id(&message_sender)
+			message_sender.encode().as_slice().derive_ethereum_address()
```

Comment:
@xiaoch05 What it the mapping factory address and message_sender sender here?

Reply:
Related: https://github.com/darwinia-network/darwinia-common/pull/1250/files#diff-c00312c658067f849ffacb5d2bfd439d44cb7305c41195906f3778e48f06837bR333-R342